.github/workflows/test.yml

```diff
@@ -264,7 +286,12 @@ jobs:
           name: "${{ github.sha }}-ac"
         if: "env.GIT_DIFF != ''"
       - name: Run tests with race detector
-        run: cat xac.txt | xargs go test -mod=readonly -timeout 15m -race -tags='cgo ledger test_ledger_mock'
+        run: cat xac.txt | xargs go test -mod=readonly -json -timeout 15m -race -tags='cgo ledger test_ledger_mock' > xac-race-output.txt
+        if: "env.GIT_DIFF != ''"
+      - uses: actions/upload-artifact@v2
```

Comment:
Again, `if: env.GIT_DIFF` might be missing here.

Reply:
Ditto